packages/devtools_app/lib/src/screens/debugger/variables.dart

```diff
@@ -114,6 +114,7 @@ Widget displayProvider(
       multiline: true,
       style: theme.fixedFontStyle,
       debuggerController: controller,
+      hoverCardTooltipController: controller.hoverCardTooltipController,
```

Comment:
nit: consider just reading this off the debuggerController if continues to be something that is hung off the debuggerController.